tests/schema/test.firefox-schemas-import.js

```diff
@@ -1202,12 +1190,9 @@ describe('firefox schema import', () => {
         .withArgs('tmp/FIREFOX_AURORA_54_BASE.tar.gz')
         .returns(mockStream);
       expect(fs.readdirSync(outputPath)).toEqual([]);
-      return fetchSchemas({ version: 54, outputPath })
-        .then(() => {
-          expect(true).toBeFalsy();
-        }, () => {
-          expect(true).toBeTruthy();
-        });
+      await expect(
+        fetchSchemas({ version: 54, outputPath })
+      ).rejects.toThrow();
```

Comment:
Same here (if we opt to make the assertions of the previous testcases to also check the error message).